### PR TITLE
Debian update

### PR DESCRIPTION
**Pull request by jaeilepp**
Updated debian version 0.10.2

### CODE REVIEW

**Comment by agramfort** [comment]
cc @yarikoptic

File: debian/changelog [hunk 1]
```diff
@@ -1,24 +1,9 @@
...
-
- -- Yaroslav Halchenko <debian@onerussian.com>  Fri, 27 Nov 2015 10:44:11 -0500
-
-python-mne (0.8.6+dfsg-2) unstable; urgency=medium
-
-  * debian/{rules.patches/changeset_b9d4d*}
-    Absorb upstream patch and specify not using HOME directory for generated
-    pre-computed data, but instead use tempdir. (Closes: #768730)
+  * NMU
+  * New upstream version
```
Comment (yarikoptic) [inline debian/changelog]: Removing previous changelog looks wrong

File: debian/copyright [hunk 2]
```diff
@@ -6,7 +6,7 @@ Files-Excluded: mne/html/*.min.js
 Comment: File AppEULA.rst refers to code that is documented in this package but not shipped with it.
 
 Files: *
-Copyright: 2010-2016, MNE-Python Developers
+Copyright: 2010-2014, MNE-Python Developers
```
Comment (yarikoptic) [inline debian/copyright]: We should go forward in time not backward

File: debian/python-mne.substvars [hunk 3]
```diff
@@ -0,0 +1,4 @@
+python:Versions=2.7
+python:Provides=python2.7-mne
+python:Depends=python (>= 2.7), python (<< 2.8), python:any (>= 2.6.6-7~), python:any
```
Comment (yarikoptic) [inline debian/python-mne.substvars]: This is a fire not to be committed

File: mne/time_frequency/psd.py [hunk 4]
```diff
@@ -168,7 +168,7 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
     else:
         time_mask = Ellipsis
 
-    data = epochs.get_data()[:, picks][..., time_mask]
+    data = epochs.get_data()[:, picks][:, :, time_mask]
```
Comment (yarikoptic) [inline mne/time_frequency/psd.py]: Debian branch should have no modifications in the upstream code. Those should be absorbed into patches

**Comment by yarikoptic** [comment]
So everything looks wrong ;-) I will check in detail when finally wake up

File: mne/time_frequency/psd.py (hunk 4 above)
Comment (jaeilepp) [inline mne/time_frequency/psd.py]: Is this really the only way to do this? This patch applies to the development version too. Changing two lines of code shouldn't be this complicated...

Comment (Eric89GXL) [inline mne/time_frequency/psd.py]: This change has already been applied to the development version:

https://github.com/mne-tools/mne-python/blob/master/mne/time_frequency/psd.py#L140

But really what should (also) be done is to change `time_mask = Ellipsis` to `time_mask = slice(None)` above. It's better anyway.

Comment (Eric89GXL) [inline mne/time_frequency/psd.py]: (and actually that's also been done in `master` https://github.com/mne-tools/mne-python/blob/master/mne/time_frequency/psd.py#L410)

**Comment by yarikoptic** [comment]
doh -- I did build 0.11 release a while back (Jan 5) but it seems forgot (seems litterally just that) to upload it to debian proper, did upload to neurodebian only http://neuro.debian.net/pkgs/python-mne.html

http://git.debian.org/?p=debian-med/python-mne.git  carries that state of packaging .  I will rebuild it now for sid to see if it fails with fresh numpy and will report back shortly

**Comment by yarikoptic** [comment]
0.11 package built just fine so I have uploaded it to Debian -- I guess all those incompatibilities were solved before 0.11 ;)  thanks and sorry about the glitch

**Comment by agramfort** [comment]
thx heaps @yarikoptic 

I am closing then.

**Pull request closed without merging**